### ISSUE TITLE
Beastmaster ready ability

### ISSUE THREAD

**Issue by blacknesscrows** [EdenServer/community#380]
When beastmaster uses "ready" there is currently no cooldown to being able to use a second "ready" ability. I.E. i can spam 3 Big scissors in quick succession if i have 3 charges. I remember a cooldown though it has been a long time since I've last used ready.

Sootle

**Comment by icariumkupo** [comment]
Incorrect. So long as the bst has a charge, he should be able to use it. No cooldown. 30 seconds to restore a charge however, with a maximum of three.

**Comment by threatsyndrome** [comment]
I think back in the day bst didn't have ready. It had something like a 2 min sic or something.

**Comment by gweivyth** [comment]
Beastmaster should not have Ready at all in this era.  They should only have Sic, which was a 1:30 cooldown that chose a TP move randomly.

**Comment by jmcmorris** [comment]
There are several new job(pet) abilities that were added after ToAU era that are a good improvement. There is balancing to be done but I think we should at least consider keeping.

**Comment by threatsyndrome** [comment]
I'm not sure if having it would be a bad thing or not. may make bst more fun. Like how drg call wyvern isnt a 2 hour cooldown.

**Comment by icariumkupo** [comment]
I agree with keeping Ready! I would even argue for Snarl to be added but I'm biased =)

**Comment by MB213** [comment]
I need to show you guys a log of a 75 jug pet attacking a high level mob or something in Dynamis / Sky. 
You would be appalled at how ineffective they are even with the updated system. 

Being objective, because I've already mostly given up on the job and switched to SMN, they actually need buffing badly. They have 280~ accuracy / attack at 75. For comparison, that's equivalent to something like what you would have with a 2 handed weapon capped at C nude with no food. Beastmaster is better than other jobs only in a few niche situations that don't particularly matter much and I think going back to Sic would narrow that niche a lot and hurt a job that's already hurting. 

Era correct, maybe, but I think you have to question what is really being achieved by going back to the old system.

**Comment by gweivyth** [comment]
> Beastmaster is better than other jobs only in a few niche situations that don't particularly matter much and I think going back to Sic would narrow that niche a lot and hurt a job that's already hurting.

Yeah that was my major gripe with it when I had it on Nasomi.  I loved leveling it up, but...then what?  You get sidelined for everything because your damage is not just a little worse, but a LOT worse than basically every other job.  I had basically all of the best gear but the only time I got to merit BST was in pet parties where honestly it just felt like we were abusing how good Pet Food Zeta biscuits were once you had Reward and Beast Affinity merits...and you couldn't join a normal merit party even if your gear was absurd because you are just Warrior-lite. 

I would love for BST to be playable here, but I just don't know what we could do to it without making the pets very imbalanced for other things.  Disposable allies are very strong, so it wouldn't be easy to balance them.  Snarl and Ready being available are good starts, but as far as buffing the pets themselves, it's just very dangerous.  Beast Affinity is already VERY strong and we used to use it to kill IT birds in Mis Coast in a BST duo due to how strong Carrie is once you're 5/5 in Beast Affinity.  If we buff the base stats of those jugs anymore, they're going to be way too strong for merit-level content.  Meanwhile BST might still be mediocre on HNMs and other endgame stuff.

A few options would be to uncap some of the jug pets, since the only ones that can actually cap out at level 75 and are easily accessible are Carrie and the Cactaur.  (Both of which sort of suck.)  With access to FunguarFamiliar at 75, BST would get a lot of help as far as being a damage dealer since Funguar is just the most useful all-around pet for killing stuff, but sadly it caps out at 65. 

We can't bring back Akamochi, which was a big reason that BST was able to actually do stuff at 75 events so what do people suggest we do here?  I'm open to hearing some other people's opinions who have played the job more recently than I have.

**Comment by icariumkupo** [comment]
I don't think you do anything. BST is a niche job that specializes in solo or other pet party situations. It's designed this way. If you make it strong enough to compare to other DD on hnms then it will become op. 

Also, I think you're making a lot of assumptions about how BST functions here. Your experiences on Nasomi do not apply. No one knows how BST affinity works bc no one has those merits. No pet should be able to break lv75 regardless of affinity merits or AF2 gloves. 75 should be the global cap for a pets.

**Comment by raventn** [comment]
Bst jug pets does need more acc to keep up. atm they have way low acc compare to smn carby exampel.
Jug pets suppose be option for best to be in party/aly to use to do more dps.

even they cap at 75 they are very low acc

**Comment by Lexxi-foxxx** [comment]
Figured I'd add to this as I played BST on retail for the better part of 7 years, at least 3 doing endgame stuff as an all BST/SMN LS. 

From what I can remember, beast affinity never raised your pet above the 75 cap. I can say this with confidence because they were always just fodder to keep mobs occupied with someone other than the SMNs. They had poor accuracy and their damage was pretty low as well, but could absorb a hit or two to buy time for SMNs to get Diabolos back out. It wasn't uncommon for SMNs to pull hate using BPs so it's not like jug pets were doing a lot of damage to keep hate. How they compare to jug pets on Eden I don't know as I am not that high yet.

If you look through the beast affinity history on wiki, it will say that it only applies to jugs that are uncapped. 5/5 BA merits opened up the HQ lizard, fly, and funguar to use at 75. During most of our time doing sky, group 2 merits were set to 3 max so putting merits into it was largely useless. Once it was increased to 5, some of the 65 cap pets saw some use, but generally never really did anything to speed up fights from the additional damage over CC. They died quicker is about the only observation I can recall which probably meant they were doing more damage than CC at least.  Pets called with call beast always had a range of 0-2 levels below the BST using the ability. Monster gloves raised this to 0-1 level difference. 

Having access to Ready is a very nice addition. Obviously not enough to warrant choosing a BST over a (insert DD job here) but its miles better than sic.  Being a private server gives the opportunity to balance jobs that SE never bothered to do (see: PUP on release, or even 3 years after release). On retail they gave the ability to skillchain with pet ready abilities. If the goal is to balance jobs, this might be something that can be considered? It would give at least a little more utility for BST, and adds (albeit minimal) damage. I don't think any of the available jugs at 75 cap would be able to do anything more than a level 1 skillchain. Snarl would also be a welcome addition allowing some added utility for BST as well. Having a THF SATA WS onto a BST was a big additional damage boost in sky that didn't result in a dead BST.

**Comment by gweivyth** [comment]
>Also, I think you're making a lot of assumptions about how BST functions here. Your experiences on Nasomi do not apply.

Oh yeah definitely, but it's the only real experience I have with the job.  I didn't have a reason to play it beyond leveling it to 70 for Maat on retail.  Nasomi Beastmaster is apparently absurdly overpowered in a lot of ways, but when I was playing there it definitely felt mediocre at best, I suppose because I wasn't blatantly abusing unintended mechanics.  While level 85 Carries were strong, they still weren't useful on HNMs, but they were pretty good for duoing and that was about it.

I think it would actually be pretty cool to give jug pets skillchain properties, that might be something to consider down the line.

**Comment by threatsyndrome** [comment]
So jug pets have ready ability and charmed pets have sic that goes off mobs TP.

**Comment by MeyiLekoh** [comment]
As much as I love Beastmaster, I feel it is overpowered. Charges replenish quickly, and you get to choose which TP move your pet uses, regardless of whether they have TP or not. But if it is judged as not unbalanced, I will not complain hahaha!

**Comment by MB213** [comment]
> As much as I love Beastmaster, I feel it is overpowered. Charges replenish quickly, and you get to choose which TP move your pet uses, regardless of whether they have TP or not. But if it is judged as not unbalanced, I will not complain hahaha!

Can you elaborate in which situations & levels you feel the job is overpowered?
I personally would like to see Beastmaster see some balancing because it was my first job to cap and I love the concept of it, however I feel for most activities outside of farming and certain BCNMs it's very weak and other pet based job options are just miles better, not to mention the other jobs. 

Since the devs are undecided on this issue, I think discussion about the jobs current strengths and weaknesses would be helpful. Right now BST is stuck in a weird no mans land and I'd like to see it join everyone else and become an actually useful job to others, rather than just a farming powerhouse.

**Comment by Balrahn** [comment]
I'm not sure how I feel about this. 

I recently did Maat as bst and thought I had a hard challenge on my hand. 
Now I haven't done Maat as bst before, but I'm pretty traumatized by Maat from era on other jobs.

Anyways: Before the fight I were able to buff up my pet pre fight with evasion boost, no TP needed.
Then I saved up 3k TP to make the evasion boost last longer. That's about when I realised this whole challenge was a joke and that I could just spam TP moves to kill Maat off quickly.

This made me feel that Ready was too OP and that it should atleast be nerfed to not have the charges and rather have the same CD as Sic.

Then I read this comment section and I'm not sure what I think is best anymore. Nerfing it could cause BST to be unfavourable in most things. Atleast in the way it's presented here.
Then again I remember a bst from my Dynamis LS that always sported a Lifedrinker Lars and he did very well.

So I don't know which direction I'd want this to go.

**Comment by godmode** [comment]
Yeah. It’s a really tough call. Which is partially why it’s been sitting for so long. BST don’t really have a spot in most end game and we don’t plan to balance it so much that they do. But I think that is what most players benchmark jobs on.

**Comment by Wade9363** [comment]
Maat fight in Retail Era on Beastmaster was among the easiest.  I was in a large pet linkshell and I don't remember anyone losing a Maat fight.

The strategy was simple even without ready.  

Sleep for TP.
Put pet on Maat.
Rampage Maat's pet, it dies.
Meele Maat until he calls a new pet.
Rampage on Maat's new pet, it dies.
Meele Maat until you win.

**Comment by greyenvy** [comment]
Removing Ready would effectively kill Beastmaster on this server.  That is not even an exaggeration either.

**Comment by mlandrum22** [comment]
Even with Ready, BST doesn't really fit in the "end game" scene -- vs HNMs and what not -- but that's fine.

Where BST currently shines (and almost exclusive BECAUSE of Ready) is in the "farming level 75-80 mobs" scene. W/o Ready, I don't think people would level BST outside of finishing Maat's cap (or just for the challenge of it).

**Comment by gweivyth** [comment]
> Yeah. It’s a really tough call. Which is partially why it’s been sitting for so long. BST don’t really have a spot in most end game and we don’t plan to balance it so much that they do. But I think that is what most players benchmark jobs on.

If that's the case, leave it the way that it is.  While it isn't great, it also isn't as useless as it was on retail. Puppetmaster also isn't very useful at endgame, but it's really good in exp and for soloing just like BST, so that niche seems to be something that people still desire. 

Hell, Beastmaster in the post-Rampage level range in ToAU camps is actually really solid, I was very surprised at how well our BST performed when I was leveling NIN.

**Comment by Balrahn** [comment]
So according to wiki. When Ready came out the pet needed 100% TP to use a charge. I'd say that should be a minimum. Right now you can just use TP moves regardless of TP.

https://ffxiclopedia.fandom.com/wiki/Ready?oldid=1012253

**Comment by mlandrum22** [comment]
Balrahn and MeyiLekoh feel Ready is OP because they're still exping and Ready actually is OP, from 25-74 when you're only fighting EM/T mobs and/or worms or other low defense mobs. It also feels OP because it's being compared to Sic, which in a lot of cases is completely useless. 

Once you're 75 and you no longer get to choose your opponents (and there are farrrrrr fewer charming options), Ready starts to feel like a necessity more so than a nice option.

While requiring 100% TP sounds like a nice compromise, this would force your pet to get hit to use Ready, which means only CC would be used since she's the only one that can actually take a hit. Also, in that version of Ready, BST had Snarl.

For instance, I recently spent a ton of time in Monastic Cavern trying to get myself a Juggernaut. In that scenario, (vs. the placeholder Orc and no mobs to charm) AmbusherAllie was the best option since she can stun Cure IV and also dispel his defensive buffs. Neither of those moves alone would make her better than CC, but combined it's a nice combo.

She couldn't tank the whole fight so we'd share hate. This means she wouldn't have the TP to use both Stun and Dispel. 

If I made her tank, she'd die too quickly making it so that I couldn't melee and then I'm back to being just a knock off SMN (since we don't have Snarl to give new pets hate AND we can't have our new pets immediately hit for some damage). 

Which then circles us back to CC being the only pet option.

Again, completely and 100% biased (I've been playing since January and have only BST leveled), but removing Ready, or even requiring 100% TP would likely just make me level SMN instead. SMN is already better than BST in basically every scenario, but BST is significantly more fun since we can actually hit the mobs.

**Comment by greyenvy** [comment]
Jesus christ, if people are really only exping on worms and the like to get BST to 75 I fear for people's ability to actually play the damn job.  Sure it changes a bunch at 75 but not understanding the basics is just terrible.  Get out there and solo some EXP and learn the mechanics.  It's actually quite enlightening!

Ready is needed to bring this job in-line with its identity.

**Comment by mlandrum22** [comment]
I didn't mean that they are only exping on worms, just that you exp off mobs with weak defense intentionally (your guide says to never fight Beetles for this reason!) and so the single target damage Ready moves feel OP because they do good damage (and are amazing at finishing off a mob after your pet died).

At 75, you aren't really using the single target damage moves, but you're choosing the pet based on the debuff you want (bubble shower + scissor guard for CC, geist wall + numbing noise for Allie, etc).

**Comment by Balrahn** [comment]
Actually the jug pets massively suck at exp. Never use them.

**Comment by mlandrum22** [comment]
At soloing? Yes, except in the scenario above where your pet just died/left and the mob has like 20% health.

When Duoing/Trioing/etc, it's best to not have all members with charmed pets. Jug pets don't go uncharmed after 3 minutes, can kill off mobs at <20% health (like above) and remove downtime of looking for new pets to charm.

**Comment by Lexxi-foxxx** [comment]
Just noticed I was getting emails regarding this discussion. When I last played I was planning to take BST to 75, but the glaring problems with the job just made it a chore to try and solo. I would have guessed this would be figured out by now, but I don't keep up with the server anymore so I have no idea if development is still going strong... 

Is there really some kind of huge issue with buffing BST, or is it just so seldom used that no one wants to bother? It's never going to be a first option for any endgame content outside of very niche situations, even if it is given snarl. If ready is available, snarl should be, but that's a different argument. I get trying to stay true to SE's version of the game, but unless you're going to replicate it (wherever possible) to the T, why not buff BST a little and make them at least usable for endgame content, even if they aren't the best. More options isn't going to break the meta.  As much as I hate nu-FFXI, being able to skillchain with a pet was a really good addition for BST. 

I did want to come back to the server at some point and continue with BST, but not until bugs got worked out

**Comment by VeilEden** [comment]
100% not opposed to BST retaining/gaining OOE moves/pets/abilities if it actually balances the job. 

Maintaining “era” shouldn’t be the goal here imo. If it took SE past our cutoff to truly balance BST and make it useful, maybe that should be our goal?

**Comment by Snugglepunk** [comment]
Not only is Ready very out of era, But we also have a super out of era buffed version. 

Ready was added in Nov 9, 2009.  It required the jug pet to have 100% TP to use the ability.
Currently on Eden you can use Ready with the Jug Pet having 0% TP.
https://www.bg-wiki.com/ffxi/Version_Update_(11/09/2009)

Ready was changed in Nov 10, 2014. "The job ability Ready will now enable the familiar to use special attacks even at less than 1000 TP as long as the charge requirements are met"
This is the current version it seems we have.  
https://forum.square-enix.com/ffxi/threads/44943

People say BST is under tuned here and needs ready to be viable and I don't believe this is the case when BST's are soloing Sky gods with jug pets :P  Also "Bst Burns" are OP exp/hr because of ready being OP. 

I believe BST also has the same issue with 0 hate like SMN has where a BST can use JA's and not gain any enmity from the mobs if they themselves have not acted on the mob.   Perhaps another that's another bug report needed or add this in with the SMN one.
https://github.com/EdenServer/community/issues/4530
